src/components/DatePicker/index.js

```diff
@@ -1,82 +1,63 @@
-import React, { Component } from 'react';
-import { DatePicker } from 'antd';
+import { Split, SplitItem, DatePicker, isValidDate, yyyyMMddFormat } from '@patternfly/react-core';
+import React, { useState } from 'react';
 import { connect } from 'dva';
-import moment from 'moment';
-import PropTypes from 'prop-types';
-import './index.less';
 
-const { RangePicker } = DatePicker;
+const mapStateToProps = state => {
+  return state;
+};
 
-@connect(({ global, datastore }) => ({
-  indices: datastore.indices,
-  selectedDateRange: global.selectedDateRange,
-}))
-class AntdDatePicker extends Component {
-  static propTypes = {
-    onChangeCallback: PropTypes.func,
-  };
+const PFDatePicker = props => {
+  const [from, setFrom] = useState('');
+  const [to, setTo] = useState('');
+
+  const toValidator = date =>
+    isValidDate(from) && date >= from ? '' : 'To date must be less than from date';
```

Comment:
Is this right? It seems that `date` here is a potential `to` date, being compared against `from`, but the error on failure is that "To date must be less than from date". Isn't that backwards? We're failing if `date` is `< from` because we expect `to` date to be the same or later than `from`.

Reply:
@dbutenhof the reason behind using `PFDatePicker` is , Patternfly is already using the keyword `DatePicker` for its component and we are also using the same in our page. So there would be a clash of same keywords if we don't use separate names.

Reply:
Fixed the `to` , `from` issue.